features/comments_and_kudos/comments_delete.feature

```diff
@@ -34,6 +38,8 @@ Feature: Delete a comment
       And I view the work "Awesome story" with comments
       And I delete the comment
     Then I should see "Comment deleted."
+      And I should not see "Comments:1"
```

Comment:
Can we use `I should not see "Comments:"`, since this stat element shouldn't show up at all?